components/javax.cache/src/main/java/org/wso2/carbon/caching/impl/internal/CachingServiceComponent.java

```diff
@@ -0,0 +1,89 @@
+/*
+*  Copyright (c) 2005-2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Lets update the license year (in all places)

Reply:
+1 Fixed